client/src/site-search/search-results.tsx

```diff
@@ -149,14 +154,15 @@ export default function SearchResults({
 
     return (
       <div>
-        <Results {...data} query={query} updateQuery={updateQuery} />
+        {/* It only makes sense to display the sorting options if anything was found */}
```

Comment:
But it also makes sense to inform about zero results and suggest to try different terms.

Reply:
The length of the results is a property of an Array that could be shown without conditions right?

Reply:
No, there's a `data.metadata.total.value` integer.

Reply:
Eh beyond where it is, we always show that, no?